textsubs.lic

```diff
@@ -890,4 +890,302 @@ TextSubs.add('Your skill in (.*) is at a zenith of enlightenment\.', 'Your skill
 # Commas when dealing with coins
 TextSubs.add('\d{1,3}(?=(\d{3})+(?!\d).*(Kronar|Dokora|Lirum))', '\&,')
 
+# Scroll subs to add guild name
+TextSubs.add('"Strange Arrow."', '"Strange Arrow(Analagous)."')
+TextSubs.add('"Burden."', '"Burden(Analagous)."')
+TextSubs.add('"Lay Ward."', '"Lay Ward(Analagous)."')
+TextSubs.add('"Manifest Force."', '"Manifest Force(Analagous)."')
+TextSubs.add('"Ease Burden."', '"Ease Burden(Analagous)."')
```

Comment:
We can take a vote on that from the user base and see.  It's not a big difference either way but if you want to keep with a previously applied styling, that's fine.

I'll work on the other corrections and submit a new version later today.